### ISSUE TITLE
zsh: compinstall is broken

### ISSUE THREAD

**Issue by nyuszika7h** [termux/termux-packages#61]
```
~> autoload -Uz compinstall
~> compinstall
compinstall:6: failed to load module: zsh/files
```

**Comment by nyuszika7h** [comment]
The problem seems to be that it's looking for `$PREFIX/usr/share/zsh` instead of `$PREFIX/share/zsh` in `$FPATH`.

**Comment by nyuszika7h** [comment]
(ignore my previous comment)

**Comment by fornwall** [comment]
The line that fails is

    zmodload -F zsh/files b:zf_ln || return 1

This means that the `files` module cannot be loaded. And looking at the `INSTALL` file in the zsh source distribution we indeed see

    For a non-dynamic zsh, the default is to compile the complete, compctl,
    zle, computil, complist, sched, parameter, zleparameter and rlimits modules
    into the shell, and you will need to edit config.modules to make any other
    modules available.

So since we build a non-dynamic zsh (due to zsh dynamic loading not working on Android), we need to explicitly enable the `files` module.

**Comment by fornwall** [comment]
@nyuszika7h This should now be fixed in the `5.2.0-1` version of the `zsh` package - update to it by executing

    apt update && apt upgrade

and let me know if there is any problem!

**Comment by nyuszika7h** [comment]
Thanks, it works now.
